.travis.yml

```diff
@@ -23,12 +25,14 @@ script:
   - test -z "$(go vet $(find ./arm/* -type d -print) | tee /dev/stderr)"
   - test -z "$(golint ./arm/... | tee /dev/stderr)"
   - go build ./arm/examples/...
-  - go test -v ./management/...
+  - if [[ $TRAVIS_GO_VERSION != 1.6* ]]; then
```

Comment:
I worry that this fix won't help people using go 1.6 who are installing our SDK as such:

`go get -u github.com/Azure/azure-sdk-for-go/...`

If we're willing to pay that price, I'm open to it.